vizro-core/src/vizro/models/_components/form/_alert.py

```diff
@@ -42,5 +42,5 @@ def build(self):
                     className="alert",
                 )
             ],
-            className="alert_container",
+            className="alert-container",
```

Comment:
I just saw that this CSS class is not used anywhere in the CSS files, so feel free to remove it.

```suggestion
```